compiler/lib/freevars.ml

```diff
@@ -53,6 +53,7 @@ let iter_instr_free_vars f i =
       f x;
       f y;
       f z
+  | Assign (_, _) -> ()
```

Comment:
```suggestion
  | Assign (x, y) -> f x; f y
```

Reply:
Actually only `y` is free. `x` is a bound variable (of the exception handler block).